repository/src/main/java/org/pentaho/platform/repository2/unified/jcr/DefaultDeleteHelper.java

```diff
@@ -371,28 +371,20 @@ public List<RepositoryFile> getAllDeletedFiles( final Session session,
     if ( isAdmin() ) {
       List<RepositoryFile> deletedFiles = new ArrayList<>();
       ITenant tenant = JcrTenantUtils.getTenant();
-      // BISERVER-14482: this user list refers to Jcr, as so, if authentication/authorization is otherwise managed
+      // (deprecated comment) BISERVER-14482: this user list refers to Jcr, as so, if authentication/authorization is otherwise managed
       // (LDAP, for instance), one has to guarantee that the current user is on this list!
       // The correct way would be to fix the list by obtaining the LDAP users, but this has the potential of getting
       // thousands of users and, consequently, thousands of deleted files!
```

Comment:
Since the comment refers to the old code would it make sense to remove it? It's usefulness would be just in navigating through github history to check the old code?

Reply:
The idea is to help the next developer find the previous change in behavior as this is a regression.

Reply:
I'm fine leaving it for that reason, I see that it's possible it may help in the future